src/js/model/getResult.js

```diff
@@ -0,0 +1,8 @@
+export default function getResult(cars) {
+  const maxScore = Math.max(...(Array.from(cars).map(it => Number(it.dataset.forwardCount))));
```

Comment:
```suggestion
  const maxScore = Math.max(cars.map(it => Number(it.dataset.forwardCount)));
```
아예 cars는 array로 변환해서 넘겨주면 어떨까요?

Reply:
model에서는 매개변수만을 받아서 결과 값을 뱉어주는 역할만 해주니 좀더 추상화되고 코드 가독성도 더 좋아진거 같아서 좋네요 !